### PR TITLE
Remove default mounts for orchestrators

### PR DESCRIPTION
**Pull request by afbjorklund**
Make k8s and nomad more similar to k3s

### CODE REVIEW

**Comment by AkihiroSuda** [comment]
Any specific reason to do so?

**Comment by jandubois** [comment]
I have no strong opinion either way. I guess having no mounts at all saves you from installing `sshfs` during first boot, but otherwise it doesn't really cost much.

On the other hand is seems unlikely that you are making use of them when you are using `k8s`. Using `hostpath` volumes is not really good practice, but given that we are running single-node dev environments, they should work reliably, and can occasionally be useful.

So I guess I have a slight preference to keep them unless there is a specific reason to drop them.

**Comment by afbjorklund** [comment]
My main goal was to make k3s and k8s more similar...

Changing whitespace and mounts were just casualties.

Similarly, I wanted Kubernetes and Nomad to be similar too.

**Comment by afbjorklund** [comment]
Why were the defaults dropped for k3s ?

**Comment by jandubois** [comment]
> Why were the defaults dropped for k3s ?

That script started out as a PR comment when I implemented provisioning scripts: https://github.com/lima-vm/lima/pull/38#issuecomment-856497148

Since it was supposed to show how provisioning works, I kept it otherwise as minimal as possible. It was not originally meant to end up in the `examples` directory.

I agree that the `mounts` should be added to `k3s` as well, if we keep them for `k8s` and `nomad`.

**Comment by afbjorklund** [comment]
I just wanted the file to be less complicated, I don't think that they will be used ? Especially now when moving `kubectl` outside, then it less likely to be doing "apply" on any file from the home directory or something

**Comment by jandubois** [comment]
> I just wanted the file to be less complicated, I don't think that they will be used ? Especially now when moving `kubectl` outside, then it is less likely to be doing "apply" on any file from the home directory or something

Yeah, the only use case I see it mounting a host directory into a container, to persist data outside the VM. I expect it to be rarely used too. As I said, I don't have a strong opinion, just a slight preference.

I'm fine with whatever @AkihiroSuda decides...

File: examples/k8s.yaml [hunk 1]
```diff
@@ -10,11 +10,6 @@ images:
     arch: "x86_64"
   - location: "https://cloud-images.ubuntu.com/focal/current/focal-server-cloudimg-arm64.img"
     arch: "aarch64"
-mounts:
-  - location: "~"
-    writable: false
-  - location: "/tmp/lima"
-    writable: true
 containerd:
```
Comment (AkihiroSuda) [inline examples/k8s.yaml:13]: Please add this note, then LGTM
https://github.com/lima-vm/lima/blob/cf42676298bb29c762ea0cb4803d4c6150147e9c/examples/k3s.yaml#L21-L22

**Comment by afbjorklund** [comment]
Ok, I will remove the mounts again and update

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Pull request merged**